lib/Core/Executor.cpp

```diff
@@ -362,33 +363,20 @@ Executor::Executor(LLVMContext &ctx, const InterpreterOptions &opts,
       DebugPrintInstructions.isSet(FILE_SRC)) {
     std::string debug_file_name =
         interpreterHandler->getOutputFilename("instructions.txt");
-    std::string ErrorInfo;
+    std::string error;
 #ifdef HAVE_ZLIB_H
     if (!DebugCompressInstructions) {
 #endif
-
-#if LLVM_VERSION_CODE >= LLVM_VERSION(3, 6)
-    std::error_code ec;
-    debugInstFile = new llvm::raw_fd_ostream(debug_file_name.c_str(), ec,
-                                             llvm::sys::fs::OpenFlags::F_Text);
-    if (ec)
-	    ErrorInfo = ec.message();
-#elif LLVM_VERSION_CODE >= LLVM_VERSION(3, 5)
-    debugInstFile = new llvm::raw_fd_ostream(debug_file_name.c_str(), ErrorInfo,
-                                             llvm::sys::fs::OpenFlags::F_Text);
-#else
-    debugInstFile =
-        new llvm::raw_fd_ostream(debug_file_name.c_str(), ErrorInfo);
-#endif
+      debugInstFile = klee_open_output_file(debug_file_name, error);
```

Comment:
unique_ptr for debugInstFile instead of raw pointer

Reply:
done